frontend/src/components/ibutsu-page.js

```diff
@@ -21,54 +21,50 @@ import { ToastContainer } from 'react-toastify';
 import { ALERT_TIMEOUT } from '../constants';
 
 
+const IbutsuPage = (props) => {
+  const {
+    eventEmitter,
```

Comment:
I'm going to follow this up to remove the eventEmitter completely, but for now it's still connected to the sidebar function call.